lib/handler/mruby.c

```diff
@@ -45,6 +45,38 @@
 
 #define FREEZE_STRING(v) MRB_SET_FROZEN_FLAG(mrb_obj_ptr(v))
 
+struct st_subreq_ostream_t {
+    h2o_ostream_t super;
```

Comment:
How about renaming the property to `_ostr_final`? That is the name we use in `struct st_h2o_http1_conn_t` and `h2o_http2_stream_t`.